integration-tests/requirements.txt

```diff
@@ -5,15 +5,24 @@
 #    pip-compile --output-file requirements.txt requirements.in
 #
 behave==1.2.5
+boto3==1.4.6
+botocore==1.6.6           # via boto3, s3transfer
 docker-py==1.10.6
 docker-pycreds==0.2.1     # via docker-py
+docutils==0.14            # via botocore
+enum34==1.1.6             # via parse-type
```

Comment:
[I've already seen this once](https://github.com/fabric8-analytics/fabric8-analytics-common/pull/130#discussion_r129033356).

You most likely use Python 2 version of pip-compile (pip-tools package), while we need Python 3.